tests/docker/configs/java_entrypoint.yaml

```diff
@@ -0,0 +1,4 @@
+schemaVersion: 2.0.0
+
+metadataTest:
+  entrypoint: ["/usr/bin/java -cp /app/io_bazel_rules_docker/testdata/libjava_bin_as_lib.jar:/app/io_bazel_rules_docker/testdata/libjava_image_library.jar:/app/io_bazel_rules_docker/../com_google_guava_guava/jar/guava-18.0.jar:/app/io_bazel_rules_docker/tests/docker/java_image.binary.jar:/app/io_bazel_rules_docker/tests/docker/java_image.binary examples.images.Binary"]
```

Comment:
you need to split lines at ' '
https://github.com/bazelbuild/rules_docker/pull/576

Reply:
got it - fixed